css/index.css

```diff
@@ -0,0 +1,39 @@
+body {
+  background-color: lightblue;
+  color: darkgreen;
+  font-family: cursive;
+}
+section {
+  margin: 20px 0;
+  padding: 10px;
+  text-align: center;
+}
+h1,
+h2,
+h3 {
+  font-size: 2em;
+  font-weight: bold;
+  color: blueviolet;
+}
+h1#name {
```

Comment:
Does this style a section?

Reply:
It targets the h1 element with the ID "name" (Vicente Flores).